src/modules/endpoint/endpoints/locations.js

```diff
@@ -15,15 +15,15 @@ function stripSlashes (path) {
 }
 
 export default new EndpointBuilder('locations')
-  .withUrl('https://cms.integreat-app.de/wp-json/extensions/v1/multisites')
+  .withRouterToUrlMapper(() => 'https://cms.integreat-app.de/wp-json/extensions/v1/multisites')
   .withMapper(json => {
     const locations = json
       .map((location) => new LocationModel({
         name: location.name,
         code: stripSlashes(location.path),
         live: location.live,
         eventsEnabled: location['ige-evts'] === '1',
-        extrasEnabled: true // todo: Adjust this in WEBAPP-64
+        extrasEnabled: true
```

Comment:
This todo is still valid until WEBAPP-64 is done, right?

Reply:
Yeah I'm also currently working on WEBAPP-64, so I thought I could just remove it here.